### PR TITLE
Lint proxies directory

### PR DESCRIPTION
**Pull request by mauriciovieira**
This PR is part of #2045 endeavor.

### CODE REVIEW

**Comment by mauriciovieira** [comment]
@apinf/developers please review

File: proxies/collection/server/publications.js [hunk 1]
```diff
@@ -1,18 +1,26 @@
-import { Proxies } from '../';
+import { Counts } from 'meteor/tmeasday:publish-counts';
+import { Meteor } from 'meteor/meteor';
+import { Roles } from 'meteor/alanning:roles';
+
+import Proxies from '../';
 
 Meteor.publish('allProxies', function () {
+  let proxies;
```
Comment (marla-singer) [inline proxies/collection/server/publications.js]: For publish function the default value must be `[]`

File: proxies/server/methods/api_umbrella.js [hunk 2]
```diff
@@ -70,6 +81,12 @@ Meteor.methods({
     }
   },
   createApiBackendOnApiUmbrella (apiBackend, proxyId) {
+    // Make sure apiBackend is a String
+    check(apiBackend, String);
```
Comment (marla-singer) [inline proxies/server/methods/api_umbrella.js]: `apiBackend` is not a String. I'll check the type and write it

Comment (brylie) [inline proxies/server/methods/api_umbrella.js]: Probably is `Object` type.

Comment (marla-singer) [inline proxies/server/methods/api_umbrella.js]: Yes, it is. @mauriciovieira Change `Strign` type to `Object`

File: proxies/server/methods/api_umbrella.js [hunk 3]
```diff
@@ -97,6 +114,12 @@ Meteor.methods({
     return response;
   },
   updateApiBackendOnApiUmbrella (apiBackend, proxyId) {
+    // Make sure apiBackend is a String
+    check(apiBackend, String);
```
Comment (marla-singer) [inline proxies/server/methods/api_umbrella.js]: @mauriciovieira Change `String` type to `Object`

File: proxies/server/methods/api_umbrella.js (hunk 2 above)
Comment (mauriciovieira) [inline proxies/server/methods/api_umbrella.js]: OK

**Pull request merged**